stable/insights-agent/templates/kube-bench/cronjob.yaml

```diff
@@ -17,8 +17,9 @@ spec:
         {{ include "job-spec-metadata" . | nindent 8 | trim }}
         spec:
           {{ include "job-template-spec" . | indent 10 | trim }}
-          {{- if or (eq (index .Values "kube-bench" "mode") "daemonsetMaster") (eq (index .Values "kube-bench" "mode") "daemonset") }}
           volumes:
+          {{ include "ssl-cert-file-volume-spec" . | indent 10 }}
+          {{- if or (eq (index .Values "kube-bench" "mode") "daemonsetMaster") (eq (index .Values "kube-bench" "mode") "daemonset") }}
```

Comment:
It looks like this line moved into the `volumes` section and out of the `spec` section.

Reply:
The primary change is that I moved the `volumes:` above this existing if statement, so I could also include the new cert-related volume. I'll further explain below, but definitely let me know if you still feel this moved something from `spec` to `volumes`.

that existing kube-bench if statement adds a volume definition, and goes on to add a container, when kube-bench is run in daemonset mode.

The content of the if statement includes `containers:`, as does the `kubebench.container.spec` define which is used in the else statement below this if.

Reply:
got it!